cime_config/tests.py

```diff
@@ -792,6 +792,14 @@
         )
     },
 
+    "e3sm_eamxx_mam4xx_v1_long_runtime" : {
```

Comment:
please stop putting "v1" in test suite names.

Reply:
Ok, I will fix it before merging.